### PR TITLE
documentation(EJ2-63820): UG documentation for MVC and Core Mention component

### PR DESCRIPTION
**Pull request by Kokila13081999**
documentation(EJ2-63820): UG documentation for MVC and Core Mention component

### CODE REVIEW

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/customization.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+{% elsif page.publishingplatform == "aspnet-mvc" %}
+
+{% tabs %}
+{% highlight razor tabtitle="CSHTML" %}
+{% include code-snippet/mention/customization/mention-char-customization/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="Data.cs" %}
+{% include code-snippet/mention/customization/mention-char-customization/mentionChar-customization.cs %}
+{% endtabs %}
+{% endif %}
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/customization.md:38]: Remove the condition compiler

Comment (saravanangn) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/customization.md:38]: Add output image for all code blocks.

**Comment by thangavele** [comment]
image

It is not correct image

File: ej2-asp-core-mvc/code-snippet/mention/accessibility/keyboard-intraction/tagHelper [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@{
+    ...
+    var data = new string[] { "Selma Rose", "Garth", "Robert", "William", "Joseph" };
```
Comment (thangavele) [inline ej2-asp-core-mvc/code-snippet/mention/accessibility/keyboard-intraction/tagHelper:3]: Why you are using the "var" here?

File: ej2-asp-core-mvc/code-snippet/mention/customization/mention-char-customization/razor [hunk 3]
```diff
@@ -0,0 +1,17 @@
+@model List<string>
+@{
+    char nameMentionChar = '@';
+}
```
Comment (thangavele) [inline ej2-asp-core-mvc/code-snippet/mention/customization/mention-char-customization/razor:4]: Should customize the mentionChar here

File: ej2-asp-core-mvc/code-snippet/mention/template/display-template/razor [hunk 4]
```diff
@@ -0,0 +1,25 @@
+@model List<string>
+@{
+    char nameMentionChar = '@';
+}
+<div id="mentionElement" placeholder="Type @Html.Raw("mention") and tag user" style="min-height: 100px; border: 1px solid #D7D7D7; border-radius: 4px; padding: 8px; font-size: 14px; width: 600px;"></div>
+
+@Html.EJS().Mention("employee").PopupWidth("250px").SuggestionCount(6).MentionChar(nameMentionChar).Target("#mentionElement").ItemTemplate("<span><span class=\"name\"> ${FirstName}</span><span class=\"city\">${Country}</span></span>").DisplayTemplate("<span>${FirstName} - ${City}</span>").DataSource(obj =>
```
[the remark this answers is not in the document]
Comment (Kokila13081999) [reply ej2-asp-core-mvc/code-snippet/mention/template/display-template/razor]: Resolved

File: ej2-asp-core-mvc/code-snippet/mention/customization/mention-char-customization/razor (hunk 3 above)
Comment (Kokila13081999) [reply ej2-asp-core-mvc/code-snippet/mention/customization/mention-char-customization/razor:4]: Resolved

File: ej2-asp-core-mvc/code-snippet/mention/accessibility/keyboard-intraction/tagHelper (hunk 2 above)
Comment (Kokila13081999) [reply ej2-asp-core-mvc/code-snippet/mention/accessibility/keyboard-intraction/tagHelper]: Resolved

**Review by thangavele — APPROVED** [review]

**Pull request merged**